src/index.css

```
@@ -8,6 +8,16 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
+.size{
+  width: 100vw;
+  height: 100vh;
+}
+
+/*Comentei esse trecho porque ainda to testando como alinhar os botões das redes sociais*/
```

```suggestion
/*Comentei esse trecho porque ainda to testando como alinhar os botões das redes sociais*/
```
Esse trecho já foi apagado porque inclui o ```.boxmodel``` que corrigi isso